examples/jax/smac/feedforward/decentralised/run_ippo_eval_intervals.py

```diff
@@ -0,0 +1,105 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Run feedforward MADQN on SMAC."""
+
+
+import functools
+from datetime import datetime
+from typing import Any
+
+import optax
+from absl import app, flags
+
+from mava.systems.jax import ippo
+from mava.utils.environments.smac_utils import make_environment
+from mava.utils.loggers import logger_utils
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string(
+    "map_name",
+    "3m",
+    "Starcraft 2 micromanagement map name (str).",
+)
+
+flags.DEFINE_string(
+    "mava_id",
+    str(datetime.now()),
+    "Experiment identifier that can be used to continue experiments.",
+)
+flags.DEFINE_string("base_dir", "~/mava", "Base dir to store experiments.")
+
+
+def main(_: Any) -> None:
+    """Example running feedforward MADQN on SMAC environment."""
+
+    # WARNING (dries): This code has not been run yet. There might still
```

Comment:
Can you remove this warning throughout Mava, please :)

Reply:
Done :)

Reply:
Thanks :)